recipes/zlib/all/conanfile.py

```diff
@@ -63,9 +63,10 @@ def _patch_sources(self):
 
         with tools.chdir(self._source_subfolder):
             # https://github.com/madler/zlib/issues/268
-            tools.replace_in_file('gzguts.h',
-                                  '#if defined(_WIN32) || defined(__CYGWIN__)',
-                                  '#if defined(_WIN32) || defined(__MINGW32__)')
+            if tools.Version(self.version) <= "1.2.11":
+                tools.replace_in_file('gzguts.h',
+                                      '#if defined(_WIN32) || defined(__CYGWIN__)',
+                                      '#if defined(_WIN32) || defined(__MINGW32__)')
```

Comment:
Since it has been fixed in 1.2.12, it could be extrated to a patch. In 1.2.11, there is only one occurrence of this pattern in gzguts.h: https://github.com/madler/zlib/blob/v1.2.11/gzguts.h#L42

Reply:
Thank you for your review.
I agree with yo.
I will create a patch file for 1.2.11.